networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc

```diff
@@ -6,6 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install External DNS Operator on cloud providers such as AWS, Azure and GCP.
+You can install External DNS Operator on cloud providers such as AWS, Azure, Infoblox and GCP.
```

Comment:
Infoblox is not a cloud provider, ExtDNS Operator which targets Infoblox is still installed in one of these 3 cloud providers. So , not need to add it here.